### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 15

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#191] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253501-236ea1f6-19ba-4971-8873-be1b43910581.png)

**Comment by i0nut02** [comment]
<img width="572" alt="Schermata 2022-02-04 alle 23 19 56" src="https://user-images.githubusercontent.com/99051485/152611077-9f659f32-c626-4ad5-b360-3a2c868cfc41.png">

**Comment by LuginiAndrea** [comment]
Analogo al #15, solo che con 2 portieri :)
